### PR TITLE
Fixing ci issues

### PR DESCRIPTION
**Pull request by davidpesce**
Fixing issues with ci

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,24 @@
 language: php
 sudo: false
 
+services:
```
Comment (ryasmi) [inline .travis.yml:4]: Do we actually need these services running? I don't think we actually hit the DB.

**Review by ryasmi — COMMENTED** [review]
I think this looks ok to me, just a comment on the services.

File: .travis.yml (hunk 1 above)
Comment (davidpesce) [reply .travis.yml:4]: It's required if we use the matrix values here:

  matrix:
    - DB=pgsql
    - DB=mysqli

And those are required by the moodle-plugin-ci

Comment (ryasmi) [reply .travis.yml:4]: Okay no worries.

**Review by ryasmi — APPROVED** [review]

**Pull request merged**